vita3k/gui/src/gui.cpp

```diff
@@ -74,9 +76,15 @@ void draw_info_message(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::TextWrapped("%s", gui.info_message.msg.c_str());
         ImGui::SetCursorPosY(WINDOW_SIZE.y - BUTTON_SIZE.y - (42.0f * SCALE.y));
         ImGui::Separator();
+        ImGui::Spacing();
+        if (ImGui::Checkbox("Show Next Time", &show_next_time))
+            ImGui::SameLine();
         ImGui::SetCursorPos(ImVec2((ImGui::GetWindowWidth() / 2.f) - (BUTTON_SIZE.x / 2.f), WINDOW_SIZE.y - BUTTON_SIZE.y - (24.0f * SCALE.y)));
-        if (ImGui::Button(emuenv.common_dialog.lang.common["ok"].c_str(), BUTTON_SIZE) || ImGui::IsKeyPressed(static_cast<ImGuiKey>(emuenv.cfg.keyboard_button_cross)))
+        if (ImGui::Button(emuenv.common_dialog.lang.common["ok"].c_str(), BUTTON_SIZE) || ImGui::IsKeyPressed(static_cast<ImGuiKey>(emuenv.cfg.keyboard_button_cross))) {
             gui.info_message = {};
+            emuenv.cfg.display_info_message = show_next_time;
+            config::serialize_config(emuenv.cfg, emuenv.cfg.config_path);
```

Comment:
revert also all of this, this have litteraly option inside settings dialog, no need add also here, you juste adding inlcude for nothing here

Reply:
I remember one user asking about how to hide the info_message and this is a bit confusing. I also think many people turn this off.

Reply:
Yes but is juste because he no know can disable option in setting ^^

Reply:
Honestly, it's just a pain to bother opening the settings and turning it off every time I download a build to test.

Reply:
?? No understund ? You using all time news folder ? 🗿

Reply:
Yes, I do so because the default settings are preferred when examining regressions.
Are there any disadvantages to this change? If there are not, then it should be merged.

Reply:
well it's just that it's useless lol
and i no want adding config file here, just revert it

Reply:
ah after look.. set this option to disable info dialog inside intial setup...
inside gui part

Reply:
> well it's just that it's useless lol and i no want adding config file here, just revert it

Please don't say that thing. info_message_dialog is completely useless for me. On the contrary, it's a hindrance.

Reply:
read my message from before, for that I told you that you could add this option to deactivate it in the initial setup, you just moved your checkbox there you know